test/test_brownian.py

```diff
@@ -247,7 +256,54 @@ def test_conditional_statistics(levy_area, use_levy):
             h_std = math.sqrt(h_var)
             normalised_hh = (h_r - h_mean) / h_std
             _, pval_h = stats.kstest(normalised_hh, stats.norm.cdf)
-            assert pval_h > 0.1 / (len(ts) - 2)
+            pvals_h.append(pval_h)
+    return jnp.array(pvals_w1), jnp.array(pvals_w2), jnp.array(pvals_h)
+
+
+@pytest.mark.parametrize("levy_area", ["", "space-time"])
+@pytest.mark.parametrize("use_levy", (False, True))
+def test_conditional_statistics(levy_area: _LA, use_levy):
+    pvals_w1, pvals_w2, pvals_h = conditional_statistics(
+        levy_area, use_levy, tol=2**-8, spacing=2**-10, spline="sqrt"
+    )
+    assert jnp.all(pvals_w1 > 0.1 / pvals_w1.shape[0])
+    if levy_area == "space-time" and use_levy:
+        assert jnp.all(pvals_w2 > 0.1 / pvals_w2.shape[0])
+        assert jnp.all(pvals_h > 0.1 / pvals_h.shape[0])
+
+
+@pytest.mark.parametrize("levy_area", ["", "space-time"])
+def test_spline(levy_area: _LA):
+    if levy_area == "":
+        splines = ("quad", "sqrt", "zero")
+    elif levy_area == "space-time":
+        splines = ("sqrt", "zero")
+    else:
+        assert False
+
+    for spline in splines:
+        pvals_w1, pvals_w2, pvals_h = conditional_statistics(
+            levy_area, use_levy=True, tol=2**-5, spacing=2**-6, spline=spline
```

Comment:
Can we parameterise this test by `use_levy=False/True`?

Reply:
I think it might be a bit pointless, but sure, it's not that expensive.